Dockerfile

```diff
@@ -510,23 +535,24 @@ RUN update-alternatives --set iptables  /usr/sbin/iptables-legacy  || true \
 ARG YAMLLINT_VERSION=1.27.1
 RUN pip3 install yamllint==${YAMLLINT_VERSION}
 
-COPY --from=dockercli     /build/ /usr/local/cli
-COPY --from=frozen-images /build/ /docker-frozen-images
-COPY --from=swagger       /build/ /usr/local/bin/
-COPY --from=delve         /build/ /usr/local/bin/
-COPY --from=tomll         /build/ /usr/local/bin/
-COPY --from=gowinres      /build/ /usr/local/bin/
-COPY --from=tini          /build/ /usr/local/bin/
-COPY --from=registry      /build/ /usr/local/bin/
-COPY --from=criu          /build/ /usr/local/bin/
-COPY --from=gotestsum     /build/ /usr/local/bin/
-COPY --from=golangci_lint /build/ /usr/local/bin/
-COPY --from=shfmt         /build/ /usr/local/bin/
-COPY --from=runc          /build/ /usr/local/bin/
-COPY --from=containerd    /build/ /usr/local/bin/
-COPY --from=rootlesskit   /build/ /usr/local/bin/
-COPY --from=vpnkit        /       /usr/local/bin/
-COPY --from=crun          /build/ /usr/local/bin/
+COPY --from=dockercli        /build/ /usr/local/cli
+COPY --from=frozen-images    /build/ /docker-frozen-images
+COPY --from=swagger          /build/ /usr/local/bin/
+COPY --from=delve            /build/ /usr/local/bin/
+COPY --from=tomll            /build/ /usr/local/bin/
+COPY --from=gowinres         /build/ /usr/local/bin/
+COPY --from=tini             /build/ /usr/local/bin/
+COPY --from=registry         /build/ /usr/local/bin/
+COPY --from=criu             /build/ /usr/local/bin/
+COPY --from=gotestsum        /build/ /usr/local/bin/
+COPY --from=golangci_lint    /build/ /usr/local/bin/
+COPY --from=shfmt            /build/ /usr/local/bin/
+COPY --from=runc             /build/ /usr/local/bin/
+COPY --from=containerd       /build/ /usr/local/bin/
+COPY --from=rootlesskit      /build/ /usr/local/bin/
+COPY --from=vpnkit           /       /usr/local/bin/
+COPY --from=containerutility /build/ /usr/local/bin/
```

Comment:
nit: could use `containerutil` (instead of `containerutility`); looks like that's _just_ short enough so that we don't have to re-flow / re-align all the other ones 😅